tensorflow-framework/src/main/java/org/tensorflow/framework/initializers/RandomNormal.java

```diff
@@ -47,39 +49,33 @@
    * creates the RandomUniform initializer
    *
    * @param tf the TensorFlow Ops
+   * @param seed the seed for random number generation. An initializer created with a given seed
+   *     will always produce the same random tensor for a given shape and dtype.
    */
-  public RandomNormal(Ops tf) {
-    this(tf, MEAN_DEFAULT, STDDEV_DEFAULT, null);
-  }
-
-  /**
-   * creates the RandomUniform initializer
-   *
-   * @param tf the TensorFlow Ops
-   * @param mean Mean of the random values to generate.
-   */
-  public RandomNormal(Ops tf, double mean) {
-    this(tf, mean, STDDEV_DEFAULT, null);
+  public RandomNormal(Ops tf, Long seed) {
+    this(tf, MEAN_DEFAULT, STDDEV_DEFAULT, seed);
   }
 
   /**
    * creates the RandomUniform initializer
    *
    * @param tf the TensorFlow Ops
-   * @param mean Mean of the random values to generate.
-   * @param stddev Standard deviation of the random values to generate.
+   * @param mean Mean of the random values to generate. Default is 0.
```

Comment:
Default doc needs to move to the place where it's defaulted. Also I'd make it refer to the default value that way the comment can't become stale (e.g. `{@link MEAN_DEFAULT}`).

Reply:
OK